### ISSUE TITLE
Unable to build using cmake 2.8.12.2

### ISSUE THREAD

**Issue by iceb0y** [tatsuhiro-t/wslay#38]
With cmake 2.8.18.2, cmake generates the following error:

```
CMake Error at CMakeLists.txt:33 (export):
  export Unknown arguments.
```

cmake 2.8.12.2 can be downloaded here: https://cmake.org/files/v2.8/cmake-2.8.12.2.tar.gz.

This version of cmake is used in ubuntu 14.04 lts.

**Comment by tatsuhiro-t** [comment]
It works with cmake 3.7.1.  Perhaps wslay uses feature not found in cmake 2.8 series?

**Comment by iceb0y** [comment]
The `CMakeLists.txt` file is declared with `cmake_minimum_required(VERSION 2.8)`, so it should support 2.8.

CMake 2.8 seems still widely used in lts distributions.

**Comment by tatsuhiro-t** [comment]
Could you make a patch to fix this so that it can work with cmake 2.8?

**Comment by tatsuhiro-t** [comment]
Fixed via #39
